packages/react/src/component-tests/IcBreadcrumb/IcBreadcrumb.cy.tsx

```diff
@@ -0,0 +1,231 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+import {
+  IcBreadcrumb,
+  IcBreadcrumbGroup,
+
+} from "../../components";
+import {
+  HAVE_BEEN_CALLED_ONCE,
+  HAVE_CLASS,
+
+ } from "../utils/constants";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { mount } from "cypress/react";
+import { SlottedSVG } from "../../react-component-lib/slottedSVG";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
+
+
```

Comment:
You've got a number of extraneous newlines around this file. I think this might fail the Prettier lint check. 
I would recommend running 'npm run prettier:fix' so it can update the formatting for you.